grafana-plugin/src/containers/AddResponders/parts/AddRespondersPopup/index.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { useState, useCallback, useEffect, useRef, useContext, FC } from 'react';
```

Comment:
Could you please rename all these `index.tsx` files to their directory's name instead? The reason is that is much easier to search for `AddRespondersPopup.tsx` then for `index.tsx` across all the files

Reply:
done! 👍